helm-charts/seldon-core-operator/templates/customresourcedefinition_v1_seldondeployments.machinelearning.seldon.io.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
   annotations:
     cert-manager.io/inject-ca-from: '{{ include "seldon.namespace" . }}/seldon-serving-cert'
     controller-gen.kubebuilder.io/version: v0.4.1
-  creationTimestamp: null
```

Comment:
Is this due to newer version of controllergen?

Reply:
I assume so.